activejob/lib/active_job/queue_adapters/sucker_punch_adapter.rb

```diff
@@ -19,11 +19,22 @@ module QueueAdapters
     #   Rails.application.config.active_job.queue_adapter = :sucker_punch
     class SuckerPunchAdapter
       def enqueue(job) #:nodoc:
-        JobWrapper.new.async.perform job.serialize
+        if JobWrapper.respond_to?(:perform_async)
+          # sucker_punch 2.0 API
+          JobWrapper.perform_async job.serialize
+        else
+          # sucker_punch 1.0 API
+          JobWrapper.new.async.perform job.serialize
+        end
       end
 
       def enqueue_at(job, timestamp) #:nodoc:
-        raise NotImplementedError, "This queueing backend does not support scheduling jobs. To see what features are supported go to http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html"
+        if JobWrapper.respond_to?(:perform_in?)
```

Comment:
looks like there is a typo - JobWrapper.respond_to?(:perform_in**?**) - should the question mark be there?